tools/astarte_dev_tool/lib/mix/tasks/astarte_dev_tool/auth/keys.ex

```diff
@@ -0,0 +1,70 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Mix.Tasks.AstarteDevTool.Auth.Keys do
+  use Mix.Task
+  alias AstarteDevTool.Commands.Auth.Keys
+
+  @shortdoc "Astarte keys generation"
+
+  @aliases []
+
+  @switches [
+    log_level: :string
+  ]
+
+  @moduledoc """
+  Astarte auth key generation.
+  If no `arg` is provided, a private key is released
+  If a private key is provided via `arg`, the linked public key is released.
+
+  ## Examples
+
+      $ mix astarte_dev_tool.auth.key
+      $ mix astarte_dev_tool.auth.key "-----BEGIN EC PRIVATE KEY-----Base64-----END EC PRIVATE KEY-----"
```

Comment:
The PR description mentions a `--from-key` that seems to be missing. Leftover?

Reply:
Ops! I've fixed PR description